src/screens/SearchScreen.tsx

```diff
@@ -2,19 +2,21 @@ import React, { useEffect, useState } from 'react';
 
 import PropTypes from 'prop-types';
 import { ScrollView } from 'react-native';
-import { Searchbar, Title } from 'react-native-paper';
+import { Button, Searchbar, Title } from 'react-native-paper';
 
 import { Tree, RootTabScreenProps } from '@types';
 import SearchCard from 'src/components/SearchCard';
 import ViewContainer from 'src/components/ViewContainer';
-import { getAllTrees, checkID } from 'src/database/firebase';
+import { getAllTrees, getActiveTrees, checkID } from 'src/database/firebase';
 
 export default function SearchScreen({
   navigation,
 }: RootTabScreenProps<'Search'>) {
   const [searchQuery, setSearchQuery] = useState('');
   const [trees, setTrees] = useState<Tree[]>([]);
-  const filtered = trees
+  const [active, setActive] = useState<boolean>(true);
+
+  let filtered = trees
     .filter(
```

Comment:
we're already filtering client side here to match search queries, do we need another read operation using a .where clause? we can do the same checking if its active or not